core/domain/exp_domain.py

```diff
@@ -705,10 +712,7 @@ def __init__(self, dest, feedback, param_changes):
         # TODO(sll): Check that this state actually exists.
         self.dest = dest
         # Feedback to give the reader if this rule is triggered.
-        self.feedback = feedback or []
-        self.feedback = [
-            html_cleaner.clean(feedback_item)
-            for feedback_item in self.feedback]
+        self.feedback = feedback
```

Comment:
Here and elsewhere, don't forget to update the docstrings when you change stuff -- looking at self.feedback.validate() doesn't make much sense if self.feedback is a Python list(str).

Reply:
done